src/preact.d.ts

```diff
@@ -98,7 +98,7 @@ declare namespace preact {
 
 	function h<PropsType>(node: ComponentFactory<PropsType>, params: PropsType, ...children: (ComponentChild | ComponentChildren)[]): JSX.Element;
 	function h(node: string, params: JSX.HTMLAttributes & JSX.SVGAttributes & { [propName: string]: any }, ...children: (ComponentChild | ComponentChildren)[]): JSX.Element;
-	function render(node: JSX.Element, parent: Element | Document, mergeWith?: Element): Element;
+	function render(node: JSX.Element | null, parent: Element | Document, mergeWith?: Element): Element;
```

Comment:
We just recently merged a PR that introduces the `ComponentChild` type:

Reply:
@marvinhagemeister Awesome! I didn't realize render also took in numbers. 

Just updated the PR!